.stack-to-nix.cache

```diff
@@ -146,3 +146,6 @@ https://github.com/input-output-hk/cardano-ledger ccb81fc69285c127c47ce622c49963
 https://github.com/input-output-hk/cardano-ledger ccb81fc69285c127c47ce622c4996317556aff42 cardano-ledger/test 0ayq8f7dgdgwbgjn7wvh7m94sa7nbgm5633hm62z3frp02l2zgfl cardano-ledger-test cardano-ledger-test.nix
 https://github.com/input-output-hk/cardano-ledger ccb81fc69285c127c47ce622c4996317556aff42 crypto 0ayq8f7dgdgwbgjn7wvh7m94sa7nbgm5633hm62z3frp02l2zgfl cardano-crypto-wrapper cardano-crypto-wrapper.nix
 https://github.com/input-output-hk/cardano-ledger ccb81fc69285c127c47ce622c4996317556aff42 crypto/test 0ayq8f7dgdgwbgjn7wvh7m94sa7nbgm5633hm62z3frp02l2zgfl cardano-crypto-test cardano-crypto-test.nix
+https://github.com/input-output-hk/iohk-monitoring-framework 9cb21a3ebf0f45bfb94ece3e6d62e613eca8b17b iohk-monitoring 1nms4gap3j6qc0gs4rk8kpqm6zdqcp5hchsd1wv459wvhfq4f20p iohk-monitoring iohk-monitoring.nix
```

Comment:
Do we even need to store `.stack-to-nix.cache` in git?

Reply:
I plead complete ignorance for all the Nix stuff.

I my opinion, there should be **no autogenerated files at all** committed to git, **ever**.

Reply:
I'll bet we need it for the benefit of CI.